src/main/groovy/com/okta/test/mock/tests/CodeFlowLocalValidationIT.groovy

```diff
@@ -243,4 +244,8 @@ class CodeFlowLocalValidationIT extends ApplicationTestRunner {
         .then()
             .statusCode(401)
     }
+
+    protected Matcher<?> loginPageMatcher() {
```

Comment:
Can you add this method to the class `OIDCCodeFlowLocalValidationIT` also?

Reply:
Can do